streaming_data_types/finished_writing_wrdn.py

```diff
@@ -0,0 +1,81 @@
+from typing import Union
+import flatbuffers
+from streaming_data_types.fbschemas.finished_writing_wrdn import FinishedWriting
+from streaming_data_types.utils import check_schema_identifier
+from typing import NamedTuple
+
+FILE_IDENTIFIER = b"wrdn"
+
+
+def serialise_wrdn(
+    service_id: str,
+    job_id: str,
+    error_encountered: bool,
+    file_name: str,
+    metadata: str = "",
+    message: str = "",
```

Comment:
Minor point, but having the default value for `metadata` and `message` as `None` might make it clearer that these are omitted from the flatbuffer, which is different to empty strings being added to the flatbuffer.

Reply:
Sure. Will fix.